pkg/controller/direct/apikeys/utils.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apikeys
```

Comment:
TBD.  We're probably going to refactor so we group reconcilers by service, so we would have 100 copies instead of 1000 copies.  But that's still a lot.

The thing we're trying to avoid is a `utils` package until we understand what we actually need in it, otherwise the utils package inevitably becomes incoherent over time.